test/extended/operators/clusteroperators.go

```diff
@@ -45,13 +54,91 @@ var _ = g.Describe("[sig-arch] ClusterOperators", func() {
 	g.BeforeEach(func() {
 		kubeConfig, err := e2e.LoadConfig()
 		o.Expect(err).ToNot(o.HaveOccurred())
-		configClient, err := configclient.NewForConfig(kubeConfig)
+		client, err := configclient.NewForConfig(kubeConfig)
 		o.Expect(err).ToNot(o.HaveOccurred())
-		clusterOperatorsList, err := configClient.ClusterOperators().List(context.Background(), metav1.ListOptions{})
+		clusterOperatorsClient = client.ClusterOperators()
+		clusterOperatorsList, err := clusterOperatorsClient.List(context.Background(), metav1.ListOptions{})
 		o.Expect(err).ToNot(o.HaveOccurred())
 		clusterOperators = clusterOperatorsList.Items
 	})
 
+	g.It("should reconcile the messages and update last transition time", func() {
```

Comment:
We're not actually expecting `lastTransitionTime` changes, as discussed [here][1].

[1]: https://github.com/openshift/enhancements/pull/1025#discussion_r799842423